src/msw/webview_ie.cpp

```diff
@@ -853,25 +857,190 @@ wxString wxWebViewIE::GetPageText() const
     }
 }
 
-void wxWebViewIE::RunScript(const wxString& javascript)
+bool wxWebViewIE::MSWSetEmulationLevel(bool defaultLevel)
+{
+    wxRegKey key(wxRegKey::HKCU, _T(REGISTRY_IE_PATH));
+    if ( key.Exists() )
+    {
+        if ( !defaultLevel )
+	    {
+            if ( !key.SetValue(wxGetFullModuleName().AfterLast('\\'), IE_EMULATION_LEVEL) )
```

Comment:
Please add a temporary variable for `wxGetFullModuleName().AfterLast('\\')`, this will avoid repeating this expression here and in the `else` branch and will allow you to give it a meaningful name, making the code more clear.